.github/ISSUE_TEMPLATE/workflow/publish.yml

```
@@ -0,0 +1,87 @@
+name: Publish package to npmjs
+
+# This workflow runs when code is pushed to `main` (i.e: when a pull request is merged)
+on:
+    push:
+        branches: [master]
+
+# Ensure that only once instance of this workflow executes at a time.
+# If multiple PRs are merged in quick succession, there will only ever be one publish workflow running and one pending.
+concurrency: ${{ github.workflow }}
+
+jobs:
+    version:
+        runs-on: ubuntu-latest
+
+        # OSBotify will update the version on `main`, so this check is important to prevent an infinite loop
+        if: ${{ github.actor != 'OSBotify' }}
+
+        steps:
+            - uses: actions/checkout@v4
```

NAB We've been using v3 in other repos, maybe we should do that here to make it consistent?
```suggestion
            - uses: actions/checkout@v3
```

---

I think we use [the version 4](https://github.com/Expensify/react-native-onyx/blob/c22acfca836437400f3bbf72a6dab16e357fde67/.github/workflows/publish.yml#L39) right now. I copied this code from the Onyx codebase